tests/LogsActivityTest.php

```diff
@@ -330,6 +330,26 @@ public function tapActivity(Activity $activity, string $eventName)
         $this->assertEquals(Carbon::yesterday()->startOfDay()->format('Y-m-d H:i:s'), $firstActivity->created_at->format('Y-m-d H:i:s'));
     }
 
+    /** @test */
+    public function it_will_not_submit_log_when_there_is_no_changes()
+    {
+        $model = new class() extends Article {
+            use LogsActivity;
+
+            protected static $submitEmptyLogs = false;
+            protected static $logAttributes = ['text'];
+            protected static $logOnlyDirty = true;
+        };
+
+        $entity = new $model();
+        $entity->save();
```

Comment:
After this save an assert should check for 0 entries.

Reply:
You mean check for 1 entries? Because when model is created, the text field changes.

Reply:
You are right. My fault - to make it more clear a real assignment to text would help.